app.css

```diff
@@ -8,8 +8,64 @@ html {
 
 body {
   margin: 0;
+  font-family: "Pavanam", sans-serif;
 }
 
 img {
   max-width: 100%;
 }
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav-bar {
+  padding: 1rem;
+  width: 100%;
+  background: #0ad9d9;
+  overflow: auto;
+}
+
+.nav-title {
+  margin: 2rem 1rem;
+  float: left;
+}
+  .nav-link {
+    font-size: 1.2rem;
+    margin: 1rem 1rem;
+    overflow: auto;
+  }
+
+  .nav-group__item {
+    float: right;
```

Comment:
Instead of floating the individual items, try floating the entire group and then just make the items `inline-blocks`